docs/hosted-orchestration/building-images.md

```diff
@@ -0,0 +1,168 @@
+---
+id: building-images
+title: Building Docker Images
+sidebar_label: Building Images
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+This document describes how to create and use custom Docker images with Sauce Labs.
+
+## Overview
+Sauce Labs supports Docker, providing you with a powerful way to orchestrate tests. In order to use Hosted Orchestration you will need to package your test code and all of its dependencies as a Docker Image and publish it to a Docker container regisry.
```

Comment:
I would consider adding to this explaining why they need to publish it (e.g.: something to the effect `publish it to a Docker container registry so that your tests can be run in our infrastructure/grid`)

Reply:
Also typo in "regisry"